aemedge/styles/styles.css

```diff
@@ -84,7 +85,8 @@
   :root {
     --udexGridGutters: var(--udexGridXLGutters);
     --udexGridMargins: var(--udexGridXLMargins);
-    --hero-margin: 246px;
+    --hero-margin--left: 246px;
```

Comment:
We are already setting `--udexGridMargins` here which is used by the here, see above?

Reply:
@mhaack We cannot set it to --udexGridMargins. We need 246px margins for XL while udexGridMargins = udexGridXLMargins for XL which is 84px

Reply:
Ok, got it.